src/main/java/Application.java

```diff
@@ -0,0 +1,59 @@
+package main.java;
+
+import java.util.Scanner;
+
+public class Application {
+
+    public static void main(String[] args) {
+
+        String greeting = "Hi, I'm just a simple java calculator and I can do addition, subtraction" +
+                ", multiplication and division of two numbers!";
+        System.out.println(greeting);
+
+        Scanner input = new Scanner(System.in);
+
+        System.out.print("Let's start with the first number - please enter it: ");
+        double value1 = input.nextDouble();
+        System.out.print("And now please choose the command (+-*/): ");
+        String command = input.next();
+        System.out.print("Now enter the second number: ");
+        double value2 = input.nextDouble();
+
+        switch (command) {
+            case "+" -> addition(value1, value2);
+            case "-" -> subtraction(value1, value2);
+            case "*" -> multiplication(value1, value2);
+            case "/" -> division(value1, value2);
+            default -> System.out.println("You entered unacceptable command.");
+        }
+
+    }
+
+    static void addition(double value1, double value2) {
```

Comment:
one of the oop principles is incapsulation. so if you can make method more 'incapsulated' (Private in this case) make it, please.